configuration/configuration.py

```diff
@@ -251,3 +251,6 @@ def _read_secret(secret_name, default = None):
 SHORT_TIME_FORMAT = environ.get('SHORT_TIME_FORMAT', 'H:i:s')
 DATETIME_FORMAT = environ.get('DATETIME_FORMAT', 'N j, Y g:i a')
 SHORT_DATETIME_FORMAT = environ.get('SHORT_DATETIME_FORMAT', 'Y-m-d H:i')
+
+# Maps provider
+MAPS_URL = environ.get('MAPS_URL', None)
```

Comment:
I know it's not completely consistent, especially with the date options, but would you mind adding this property before MAX_PAGE_SIZE, so that the alphabetical order can remain intact where it still is?